examples/goto.py

```diff
@@ -20,15 +20,21 @@ async def run():
             print("Global position estimate ok")
             break
 
+    print("Fetching Absolute Height of Terrain....")
```

Comment:
It feels like you want the [`home position`](https://github.com/mavlink/MAVSDK-Proto/blob/master/protos/telemetry/telemetry.proto#L18) here. Would that make sense?

Reply:
That is not exactly the home position, actually i was just wanted to get the home altitude (amsl). We can change it to something like "Fetching AMSL", i did this as it makes it easier to use goto_location as we can then just add our required drone altitude to the obtained amsl and provide it to the function.

Reply:
I have changed the print statement to better reflect the reason for fetching the postion() update..

Reply:
What I mean is that the home position gives you the altitude at which you took off. So if it's about having an estimation of the absolute height of the terrain, it seems reasonable to me. It should do the same as what you are doing now, unless the drone is already flying (in which case the home altitude would be what you want).

Or is the home altitude not correct in this case?

Reply:
Great, I got your point. Actually I misunderstood "position updates" with "home position update"  and yes I was trying to do the same thing and yes it makes more sense to use home position here. Thanks for your suggestion. I will make the changes.

Reply:
I have made the changes, please go through them. Thanks.